src/CardForm/CVVInput/CVVInput.js

```diff
@@ -3,31 +3,64 @@ import classNames from 'classnames';
 import React from 'react';
 import TextInput from '../TextInput';
 
-const CVVInput = ({ cvvTooltip, className, ...props }) => {
-  const textInputClassName = classNames('cvvInput', className);
-
-  return (
-    <TextInput
-      className={textInputClassName}
-      ariaDescribedBy="cvvInput-Label-Tooltip"
-      {...props}
-    >
-      <span className="cvvInput-Icon-Tooltip" />
-      <span
-        id="cvvInput-Label-Tooltip"
-        className="cvvInput-Label-Tooltip"
-        role="tooltip"
+class CVVInput extends React.Component {
```

Comment:
Why going back from functional to class component?

Reply:
I had to implement an internal state and this component is used in opp-script, which do not support hooks yet.